### ISSUE TITLE
SUB - 2.1.2 - No Keyboard Trap - Check why WCAG2ICT does not have Non-Interference section

### ISSUE THREAD

**Issue by TanyaVanWorkum** [w3c/matf#183]

**Comment by JJdeGroot** [comment]
Thanks for the clarification @maryjom 
We have discussed your comment and the conformance section in our  MATF meeting on 5 November 2025.

<details>
<summary>Minutes for 5 November 2025</summary>

[Source](https://www.w3.org/2025/11/05-matf-minutes.html#1da7)

~~~text
<JJ> Poll: Leave non-interference section from WCAG2Mobile just like WCAG2ICT?

<Tanya> +1

<pauljadam> +1

<shoobe01> +1

<quintinb> +1

<JJ> Poll results: 4x +1
~~~
</details>

Poll result: WCAG2Mobile will not include a non-interference section like WCAG2ICT.
